src/components/views/dialogs/MessageEditHistoryDialog.js

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2019 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import MatrixClientPeg from "../../../MatrixClientPeg";
+import { _t } from '../../../languageHandler';
+import sdk from "../../../index";
+import {wantsDateSeparator} from '../../../DateUtils';
+import SettingsStore from '../../../settings/SettingsStore';
+
+export default class MessageEditHistoryDialog extends React.Component {
+    static propTypes = {
+        mxEvent: PropTypes.object.isRequired,
+    };
+
+    componentWillMount() {
+        this.loadMoreEdits = this.loadMoreEdits.bind(this);
```

Comment:
Can you use [property initializer](https://github.com/matrix-org/matrix-react-sdk/blob/develop/code_style.md#react) style instead?

```js
loadMoreEdits = async (backwards) => {

}
```